src/components/AlertBox.tsx

```
@@ -0,0 +1,73 @@
+    padding: '0.5rem',
+    color: 'white',
+    width: '100%',
+  }
+  return (
+    <div style={{ padding: '2px', width: '100%' }}>
+      <div style={style}>
+        <FlexRow gap='0.5rem'>
+          <AlertBoxIcon>
+            <MegaphoneIcon animated />
+          </AlertBoxIcon>
+          <div dangerouslySetInnerHTML={{ __html: html }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)